test/integration_tests/hubert_test.py

```diff
@@ -0,0 +1,26 @@
+import torchaudio
```

Comment:
Note to reviewer: This is an integration test, which is not a part of unit test as it depends on external resources. (network and the actual pre-trained model)

We want to run this test suite occasionally (say once a day on `main` branch), which I have not set up yet but will follow up.